packages/create-sitecore-jss/src/templates/nextjs/src/EditingComponentPlaceholder.tsx

```diff
@@ -0,0 +1,14 @@
+import {
+  RouteData,
+  EDITING_COMPONENT_PLACEHOLDER,
+  EDITING_COMPONENT_ID,
+  Placeholder,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+const EditingComponentPlaceholder = ({ rendering }: { rendering: RouteData }): JSX.Element => (
```

Comment:
@illiakovalenko Can we move this to a base package?

Reply:
Good idea, done